code/__HELPERS/roundend.dm

```diff
@@ -269,6 +270,43 @@
 	if(CONFIG_GET(flag/automapvote))
 		SSvote.initiate_vote("map", "BeeBot", forced=TRUE, popup=TRUE) //automatic map voting
 
+	if(eorg_clients)
+		for(var/client/C in eorg_clients)
+			if(C) //This check is fucking awful. This also needs to block until the round ends.
+				var/turf/dest = pick_n_take(GLOB.eorg_waypoints)
+				if(!dest)
+					dest=GLOB.eorg_default
+				//This is gonna be fucking ugly but deal with it.
+				//This proc is going to be a fuckload of repeated istypes and I'd like to not waste that many cycles.
+				switch(C.mob?.type)
+					//Already Alive
+					if(/mob/living/carbon/human)
+						var/mob/living/carbon/human/prepper = C.mob
+						prepper.equipOutfit(/datum/outfit/job/assistant)
+						C.mob.forceMove(dest)
+						continue
+					//Not in a living mob
+					if(/mob/dead/observer)
+						var/mob/living/carbon/human/jonesy = new(dest)
+						jonesy.equipOutfit(/datum/outfit/job/assistant)
+						jonesy.client = C
+						continue
+					if(/mob/dead/new_player)
+						continue //Don't touch new players.
+					//Give AIs a reward for suffering through the round being an AI.
+					if(/mob/living/silicon/ai)
+						var/mob/living/silicon/robot/modules/syndicate/borgie = new(dest)
```

Comment:
This seems a lot more powerful considering nobody in the arena gets flash protection or any weapons at all

Reply:
(balance issue rather than an actual issue)